### DISCUSSION TITLE
I can't use dash to panel. How can i fix it ?

### DISCUSSION THREAD

**Discussion by hunter211277** [home-sweet-gnome/dash-to-panel#1956] [Q&A] [answered]
Before reinstalling the system, I could still use dash to panel, but after reinstalling the system, it doesn't work anymore

My computer configuration is as follows: Ubuntu 22.04.2 LTS, GNOME 42.9, NVIDIA-SMi 535.86.05. When I switch OFF to ON on the dash to panel webpage, there is no change on the desktop. When I refresh the webpage, it is still OFF. What should I do?

**Comment by mcbluefire** [comment]
Similar issue.  Had it running just fine for a couple days, then today it just turns off no matter what I do.

Tried using git to pull build from the 22nd of September and did a make install.  dconf-editor shows "No schema found" for the keys.  As expected extensions.gnome.org shows that I'm running an outdated version so cannot launch the config.  Also not available through gnome-tweaks.

Ubuntu 22.04.3 LTS,  Gnome 42.9, nvidia 535.104.05.

**Comment by mcbluefire** [comment, marked as answer]
I finally got dock to panel working again on my rig.  Had to just give up and reset gnome to defaults:
conf reset -f /org/gnome/

This resets all preferences from background picture, dock config, appearance styles.  But well worth it to get rid of whatever broke in dconf.
